Looms.PoS/Looms.PoS.Application/Features/Reservation/Commands/CreateReservation/CreateReservationRequestValidator.cs

```diff
@@ -0,0 +1,77 @@
+using FluentValidation;
+using Looms.PoS.Application.Models.Requests.Reservation;
+using Looms.PoS.Application.Utilities.Validators;
+using Looms.PoS.Application.Utilities;
+using Looms.PoS.Domain.Interfaces;
+using Looms.PoS.Domain.Exceptions;
+
+namespace Looms.PoS.Application.Features.Reservation.Commands.CreateReservation;
+
+public class CreateReservationRequestValidator : AbstractValidator<CreateReservationRequest>
+{
+    private readonly IServicesRepository _servicesRepository;
+    private readonly IReservationsRepository _reservationsRepository;
+    public CreateReservationRequestValidator(IServicesRepository servicesRepository, IReservationsRepository reservationsRepository)
+    {
+        _servicesRepository = servicesRepository;
+        _reservationsRepository = reservationsRepository;
+
+        RuleFor(x => x.CustomerId)
+            .MustBeValidGuid()
+            .MustAsync(async (request, customerId, cancellation) =>
+                {
+                    if (Guid.TryParse(customerId, out var customerGuid))
```

Comment:
.Cascade(CascadeMode.Stop) instead of this if

Reply:
so is it better for me to create extra methods instead of this?

Reply:
what extra methods?

Reply:
like everything that is in if just make a method out of it and use it like that

Reply:
sure, but the if itself is not needed

Reply:
changed it